forgerock-openbanking-uk-aspsp-common/src/main/java/com/forgerock/openbanking/common/model/openbanking/persistence/payment/converter/v3_1_5/ConsentStatusCodeToResponseDataStatusConverter.java

```diff
@@ -248,4 +249,13 @@
         }
     }
 
+    public static OBWriteDomesticConsentResponse5Data.ReadRefundAccountEnum toOBWriteDomesticConsentResponse5DataReadRefundAccount(FRReadRefundAccount frReadRefundAccount){
```

Comment:
Introduced on 3.1.5 converter a method to handling `ReadRefundAccount` enum for API responses.